.pre-commit-config.yaml

```diff
@@ -9,29 +9,25 @@ repos:
     - id: end-of-file-fixer
     - id: trailing-whitespace
 
--   repo: https://github.com/psf/black-pre-commit-mirror
-    rev: 23.11.0
-    hooks:
-    -   id: black
-        exclude: ^4-tools/is_this_pep8.py
-
 -   repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: "v0.1.5"
+    rev: "v0.6.2"
     hooks:
       - id: ruff
         args: ["--fix", "--show-fixes"]
         exclude: 4-tools/is_this_pep8.py
+      - id: ruff-format
+        exclude: 4-tools/is_this_pep8.py
```

Comment:
A better idea would be to exclude this file once in `ruff.toml`, not twice in the `pre-commit` configuration.